src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
+ * Class for Deadline tasks.
+ */
+public class Deadline extends Task {
+    private LocalDate deadline;
+
+    /**
+     * Constructor for Deadline object.
```

Comment:
```suggestion
     * Constructor for Deadline object.
 
```
And for other methods as well. There should be a newline between description and params.

Reply:
Yes, I agree with this.

Reply:
Will note that when reworking my documentation, thanks for pointing it out!